web/explorer/src/components/UploadOptions.vue

```diff
@@ -65,9 +71,17 @@ import FloatLabel from "primevue/floatlabel";
 import InputText from "primevue/inputtext";
 import Button from "primevue/button";
 
+import { useRouter } from "vue-router";
+const router = useRouter();
+
 const loadURL = ref("");
+const isBundle = import.meta.env.MODE === "bundle";
+
+defineEmits(["load-from-local", "load-from-url"]);
 
-defineEmits(["load-from-local", "load-from-url", "load-demo-static", "load-demo-dynamic"]);
+const dynamicURL =
+    "https://raw.githubusercontent.com/mandiant/capa-testfiles/master/rd/0000a65749f5902c4d82ffa701198038f0b4870b00a27cfca109f8f933476d82.json";
+const staticURL = "https://raw.githubusercontent.com/mandiant/capa-testfiles/master/rd/al-khaser_x64.exe_.json";
```

Comment:
I think this works ok.

Alternatively, we can move the files into the public directory within GH Actions and serve the content from GH Pages. I'm not sure if there's a good reason to do one or another, so don't need to change it.